topgun/k8s/k8s_suite_test.go

```diff
@@ -139,12 +269,13 @@ func helmDeploy(releaseName, namespace, chartDir string, args ...string) *gexec.
 
 func helmInstallArgs(args ...string) []string {
 	helmArgs := []string{
-		"--set=web.livenessProbe.failureThreshold=3",
-		"--set=web.livenessProbe.initialDelaySeconds=3",
-		"--set=web.livenessProbe.periodSeconds=3",
-		"--set=web.livenessProbe.timeoutSeconds=3",
 		"--set=concourse.web.kubernetes.keepNamespaces=false",
+		"--set=concourse.worker.bindIp=0.0.0.0",
 		"--set=postgresql.persistence.enabled=false",
+		"--set=web.resources.requests.cpu=500m",
+		"--set=worker.readinessProbe.httpGet.path=/",
```

Comment:
without this, our checks for "is this worker ready" would simply always resolve to "true", regardless of that being true or not (as there is no default `readinessProbe` for the worker: see https://github.com/helm/charts/blob/ed71e4a2bde403c73efab8b4d601b3fd828b75e5/stable/concourse/values.yaml#L1574-L1577)

Reply:
this seems like a really good addition